CHANGELOG.md

```diff
@@ -40,7 +40,7 @@
 - Removed `privacy` route and codes related to it
 - Improved general testing infrastructure
 - Added test cases for the index function in `Controllers/Course`
-- Added test cases for the depts function in `Controllers/Course`
```

Comment:
In general you should not delete old Changelog entries, even if they are no longer applicable.